dhis-2/dhis-services/dhis-service-analytics/src/test/java/org/hisp/dhis/analytics/table/JdbcEventAnalyticsTableManagerTest.java

```diff
@@ -574,4 +558,24 @@ private String getYearsQuery( Program p, AnalyticsTableUpdateParams params )
             " and psi.executiondate is not null and psi.deleted is false " + (params.getFromDate() != null
                 ? "and psi.executiondate >= '" + DateUtils.getMediumDateString( params.getFromDate() ) + "'" : "" );
     }
+
+    private String getYearQueryForCurrentYear( Program p, boolean withExecutionDate )
```

Comment:
Use `program` as variable name, avoid one-character argument names for readability.